src/System.DirectoryServices.AccountManagement/tests/PrincipalContextTests.cs

```diff
@@ -288,7 +288,7 @@ public void UserName_GetWhenDisposed_ThrowsObjectDisposedException()
             Assert.Throws<ObjectDisposedException>(() => context.UserName);
         }
 
-        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer), nameof(PlatformDetection.IsNotWindowsIoTCore)))]
```

Comment:
Why did you change this to ConditionalFact?  How does that work with arguments and InlineDatas?

Reply:
Trying to do two things at once.